test/transcoding/enqueue_kernel.cl

```diff
@@ -40,8 +41,6 @@
 // CHECK-LLVM: @__block_literal_global = internal addrspace(1) constant [[BlockTy1]] { i32 12, i32 4, i8 addrspace(4)* addrspacecast (i8* null to i8 addrspace(4)*) }, align 4
 // CHECK-LLVM: @__block_literal_global.1 = internal addrspace(1) constant [[BlockTy1]] { i32 12, i32 4, i8 addrspace(4)* addrspacecast (i8* null to i8 addrspace(4)*) }, align 4
 
-typedef struct {int a;} ndrange_t;
```

Comment:
It would be nice if it is possible to leave this as-is: `opencl-c.h` which is included by `-finclude-default-header` is pretty big and will slow down the test execution

Reply:
NULL is defined in opencl-c.h
So adding a NULL define here?

Reply:
Yep, there is no problem with copying definition of NULL here

Reply:
done